examples/advanced-pytorch/README.md

```diff
@@ -69,3 +69,11 @@ but this can be changed by removing the `--toy` argument in the script. You can
 The `run.sh` script starts processes in the background so that you don't have to open eleven terminal windows. If you experiment with the code example and something goes wrong, simply using `CTRL + C` on Linux (or `CMD + C` on macOS) wouldn't normally kill all these processes, which is why the script ends with `trap "trap - SIGTERM && kill -- -$$" SIGINT SIGTERM EXIT` and `wait`. This simply allows you to stop the experiment using `CTRL + C` (or `CMD + C`). If you change the script and anything goes wrong you can still use `killall python` (or `killall python3`) to kill all background processes (or a more specific command if you have other Python processes running that you don't want to kill).
 
 You can also manually run `python3 server.py` and `python3 client.py --client-id <ID>` for as many clients as you want but you have to make sure that each command is run in a different terminal window (or a different computer on the network).
+
+
+## About Differential Privacy
+If you want to achieve differential privacy, please use `alexnet` model, because efficientnet is particularly affected by noise. 
+
+If you add a little noise when using `efficientnet`, the loss will be Nan. This is issue:https://github.com/adap/flower/issues/2342
+
+Therefore, if you want to achieve differential privacy, please use `alexnet`
```

Comment:
I'm not sure if we should mention this here since this example is not about DP... I would remove this.

Reply:
I just want to tell other users that if they want to use the example of advanced-pytorch to achieve differential privacy, please do not use efficientnet

Reply:
We'll introduce new DP examples soon. I would suggest removing this just because this example is not about DP but to show how to use more features of Flower (compared to [quickstart-pytorch](https://github.com/adap/flower/tree/main/examples/quickstart-pytorch)).

Reply:
ok,that is sound good,i will close this pr

Reply:
oh no need to close it

Reply:
it's good to include both models. My comment was more about just including that section explaining the DP issue with efficientnet

Reply:
okok,i got it,i will remove the  the parts related to differential privacy in readme